client/js/table-view.js

```diff
@@ -78,15 +80,43 @@ class TableView {
 		this.sheetBodyEl.appendChild(fragment);
 	}
 
+	renderTableFooter() {
+		const fragment = document.createDocumentFragment();
+		let total = {};
+
+		for (let col = 0; col < this.model.numCols; col++) {
+			for (let row = 0; row < this.model.numRows; row++) {
+				const position = {col : col , row: row};
+				const value = this.model.getValue(position);
+				if(value && !isNaN(value)){
+					const toNum = parseFloat(value, 10);
```

Comment:
Your variable names should describe the thing (in the case of objects or numbers, etc.) or the function (in the case of functions) they represent. "toNum" suggests a function that is doing something, but this is actually just a number.